infrastructure/src/main/java/org/corfudb/infrastructure/log/StreamLogFiles.java

```diff
@@ -1297,4 +1300,34 @@ public static int getChecksum(int num) {
             return hasher.putInt(num).hash().asInt();
         }
     }
+
+    /**
+     * Estimate the size (in bytes) of a directory.
+     * From https://stackoverflow.com/a/19869323
+     */
+    long estimateSize(Path directoryPath) {
```

Comment:
Why not use Files.size() ?

Reply:
`Files::Size` doesn't actually return what I want (i.e. total size of all the logs in /log). We have to traverse the directory and sum the log file sizes.

Reply:
https://commons.apache.org/proper/commons-io/javadocs/api-2.5/org/apache/commons/io/FileUtils.html

`sizeOf` ?

Reply:
`sizeOf` calls `sizeOfDirectory` the reason I didn't use the latter is because it can throw an exception 
if the directory is concurrently modified. https://stackoverflow.com/a/11285406